hazelcast/src/test/java/com/hazelcast/ringbuffer/impl/RingbufferContainerSerializationTest.java

```diff
@@ -54,7 +54,7 @@
  * This test also forces a delay between the serialization and deserialization. If a ringbuffer is configured
  * with a ttl, we don't want to send over the actual expiration time, because on a different member in the
  * cluster, there could be a big time difference which can lead to the ringbuffer immediately cleaning or cleaning
- * very very late.
+ * very late.
```

Comment:
I half intended to leave this doubling in. But I guess I forgot when I was running through the duplicates.